src/entries/popup/hooks/useInfiniteTransactionList.ts

```diff
@@ -31,9 +31,13 @@ export const useInfiniteTransactionList = ({
 }: UseInfiniteTransactionListParams) => {
   const { currentAddress: address } = useCurrentAddressStore();
   const { currentCurrency: currency } = useCurrentCurrencyStore();
-  const { getPendingTransactions } = usePendingTransactionsStore();
+  const { pendingTransactions: storePendingTransactions } =
+    usePendingTransactionsStore();
   const [manuallyRefetching, setManuallyRefetching] = useState(false);
-  const pendingTransactions = getPendingTransactions({ address });
```

Comment:
calling `getPendingTransactions` is not allowing `pendingTransactions` to update inside the component since is not listening to state